spec/command/apply_template_spec.rb

```diff
@@ -135,7 +135,7 @@
   end
 
   context "when workload already exists" do
-    let!(:app) { dummy_test_app("with-rails", create_if_not_exists: true) }
+    let!(:app) { dummy_test_app("rails", create_if_not_exists: true) }
```

Comment:
Renamed a lot of the apps so that the names are shorter, because Control Plane has a limit of 64 characters.